data/json/items/tools.json

```diff
@@ -7419,5 +7420,19 @@
     "bashing": 6,
     "material": "steel",
     "use_action": { "type": "weigh_self", "max_weight": 150000 }
+  },
+  {
+    "id": "acetylene_machine",
+    "type": "TOOL",
+    "name": "acetylene-gas machine",
+    "symbol": "&",
+    "color": "grey",
```

Comment:
```suggestion
    "color": "light_gray",
```
or
```suggestion
    "color": "dark_gray",
```

Reply:
Mayhaps I am an idiot.  light_grey I choose you!